docs/.vuepress/theme/styles/index.styl

```diff
@@ -474,8 +460,21 @@ kbd {
 
 /* Code field styling */
 
-@media (max-width: $MQMobileNarrow) {
+@media (max-width: $extraLarge) {
   .theme-default-content div[class*="language-"] {
     margin initial
   }
 }
+
+/* Handsontable drawing on the introduction page */
+
+.handsontable-drawing {
+  width: 90%;
+  margin: 0 auto;
+  
+  p {
+    &:last-child {
+      display: none;
+    }
+  }
+}
```

Comment:
I like the image! (every feature page should have an image like that). It'd be good if all images matched the style of our own [brand guidelines](https://handsontable.com/static/resources/ModBranding/handsontable-brand-guidelines_v1.pdf), though (just a thought)

Reply:
I think that's rather a gap in our branding guidelines ;) but thanks, I also like the style of that drawing.